configs/sst_conversions.yaml

```diff
@@ -0,0 +1,30 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: RHC Worker Bash
+  description: Remote Host Configuration (rhc) worker for executing bash scripts on hosts managed by Red Hat Insights.
+  maintainer: sst_conversions
+
+  packages:
+    - rhc-worker-bash
+  
+  labels:     
+  - c9s # TODO(r0x0d): Verify if this is the correct label for RHEL 7
```

Comment:
Is this only for EL7, or do you expect this to come in to EL8/EL9/EL10 too?

Reply:
Only for EL7 for now.